apps/src/lib/util/AnalyticsConstants.js

```diff
@@ -142,9 +142,11 @@ const EVENTS = {
   VIDEO_PAUSED: 'Video Paused',
   VIDEO_ENDED: 'Video Played To Completion',
 
+  // congrats and certificates
   BATCH_CERTIFICATES_PAGE_VIEWED: 'Batch Certificates Page Viewed',
-  TEACHER_HOC_CONGRATS_PAGE_VISITED:
-    'Teacher Hour of Code Congrats Page Visited ',
```

Comment:
I assume the screenshot in the description is from before this string was updated?

Reply:
yep -- I put the new event in [this message](https://github.com/code-dot-org/code-dot-org/pull/56679#discussion_r1497511377)